ports/glutin/window.rs

```diff
@@ -84,6 +84,7 @@ pub struct Window {
 
     mouse_pos: Cell<Point2D<i32>>,
     key_modifiers: Cell<KeyModifiers>,
+    pub current_url: RefCell<Option<String>>,
```

Comment:
Sorry about that, still learning Rust and realized too that `pub` not needed. Thank you!